cli/src/main/kotlin/OrtMain.kt

```diff
@@ -71,7 +71,7 @@ data class GlobalOptions(
     val forceOverwrite: Boolean
 )
 
-class OrtMain : CliktCommand(name = ORT_NAME, invokeWithoutSubcommand = true) {
+object OrtMain : CliktCommand(name = ORT_NAME, invokeWithoutSubcommand = true) {
```

Comment:
Does this change work well together with the succeeding commit wrt. to our log function (see below)?
I wonder what `this::class.java` evaluates when it's an object. I guess it's an anonymous class but haven't checked.

```
val Any.log: KotlinLogger
    inline get() = loggerOfClass.getOrPut(this::class.java) {
        this::class.qualifiedName?.let { name ->
            require(name.startsWith("org.ossreviewtoolkit.")) {
                "Logging is only allowed on ORT classes, but '$name' is used."
            }
        }

        loggerOf(this::class.java)
    }
```